src/common/webviewEvents.ts

```diff
@@ -1,10 +1,11 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-export type WebviewEvent = "getState" | "getUrl" | "ready" | "setState" | "telemetry" | "websocket";
+export type WebviewEvent = "getState" | "getUrl" | "openInEditor" | "ready" | "setState" | "telemetry" | "websocket";

```

Comment:
Why not using websocket to pass the message?

Reply:
This is the type of message being sent. The websocket type is used for when we are passing the devtools CDP json payload that needs to be forwarded over the websocket. 
The new 'openInEditor' type will only be used to tell the extension to open an editor.